apis/v1alpha1/cluster_types.go

```diff
@@ -106,6 +106,9 @@ type CrdbClusterSpec struct {
 	// (Optional) If specified, the pod's scheduling constraints
 	// +optional
 	Affinity *corev1.Affinity `json:"affinity,omitempty"`
+	// (Optional) If specified, the pod's topology spread constraints
+	// +optional
+	TopologySpreadConstraints []corev1.TopologySpreadConstraint `json:"topologySpreadConstraints"`
```

Comment:
could you add `omitempty` to the json tag like affinity has?

Reply:
Done